code/game/objects/items/weapons/storage/bible.dm

```diff
@@ -52,47 +56,57 @@
 		name = book_name
 		SSticker.Bible_name = book_name
 
-	var/new_book_style = input(user,"Which bible style would you like?") in list("Bible", "Quran", "Scrapbook", "Creeper", "White Bible", "Holy Light", "Atheist", "Tome", "The King in Yellow", "Ithaqua", "Scientology", "the bible melts", "Necronomicon")
+	var/new_book_style = input(user,"Which bible style would you like?") in list("Generic", "Bible", "White Bible", "Quran", "Torah", "Holy Light", "Tome", "Scroll", "The King in Yellow", "Ithaqua", "Trinary", "Stars", "Scrapbook", "Atheist", "Necronomicon")
 	switch(new_book_style)
-		if("Quran")
-			icon_state = "quran"
-			item_state = "quran"
-		if("Scrapbook")
-			icon_state = "scrapbook"
-			item_state = "scrapbook"
-		if("Creeper")
-			icon_state = "creeper"
-			item_state = "creeper"
+		if("Bible")
+			icon_state = "white"

```

Comment:
Shouldn't this be using regular black bible sprite?

Anyway, removing extra bible sprites seems a little unnecessary, as they weren't used for much harm other than a few creative fun religons ala glorified book club, and there isn't much malice in that. We shouldn't punish players for being a little creative (and anything too creative can be solved with a BWOINK anyway)